README.md

```diff
@@ -53,6 +23,10 @@ To install the latest release without Visual Studio, download the [.NET SDK nigh
 
 See [what's new with the C# and VB compilers](https://github.com/dotnet/roslyn/wiki/Changelog-for-C%23-and-VB-compilers).
 
+**Pre-release builds** are available from the following public NuGet feeds: 
+- [Compiler](https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet5): `https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json`
```

Comment:
Should we mention that to get rolling Compiler builds they would need to reference the dotnet-tools feed instead? dotnet5 seems to track the latest sdk preview.

Reply:
Changed the Compiler link to point to dotnet-tools and added .NET SDK as a separate feed.